jest.config.js

```diff
@@ -10,9 +13,19 @@ module.exports = {
 		'@pubpub/deposit-utils/datacite':
 			'<rootDir>/node_modules/@pubpub/deposit-utils/dist/cjs/datacite/index.js',
 	},
-	globalSetup: '<rootDir>/stubstub/global/setup.js',
-	globalTeardown: '<rootDir>/stubstub/global/teardown.js',
+	transform: {
+		'^.+\\.[tj]sx?$': [
+			'ts-jest',
+			{
+				diagnostics: false,
```

Comment:
I disabled typechecking for tests, because `npm run test` already runs eslint which also does typechecking.
Remove that line to re-enable typechecking

Reply:
Oooh, does this speed up the tests at all?

Reply:
hmmm not sure honestly! let me check